code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -70,72 +84,79 @@
 	dead_mob_list -= R
 	living_mob_list |= R
 	R.notify_ai(ROBOT_NOTIFICATION_NEW_UNIT)
-	return 1
-
+	return TRUE
 
+/*
+ * VTEC Module
+ */
+#define VTEC_MOVE_DELAY_REDUCTION 0.5
```

Comment:
This is needlessly noisy. Change back to in-line number literals or add `var/move_delay_reduction = 0.5` to the type.